src/storage.ts

```diff
@@ -1338,17 +1340,35 @@ export class Storage extends Service {
         }
 
         const itemsArray = resp.items ? resp.items : [];
+        const unreachableArray = resp.unreachable ? resp.unreachable : [];
+
         const buckets = itemsArray.map((bucket: BucketMetadata) => {
           const bucketInstance = this.bucket(bucket.id!);
           bucketInstance.metadata = bucket;
+          bucketInstance.unreachable = false;
```

Comment:
If you make `unreachable` not optional, we can remove this as it should default to false.